src/live-share/test/index.ts

```diff
@@ -0,0 +1,24 @@
+//
```

Comment:
Had to create a separate `Live SHare` tests config since the tests are taking longer than other  `Peacock` tests due to the need to share a workspace.